corpus/segments.py

```diff
@@ -49,7 +50,16 @@ def run(self):
         for idx, segments in enumerate(chunks(all_segments, self.num_segments)):
             with open(self.out_single_segment_files[idx + 1].get_path(), "wt") as segment_file:
                 for segment in segments:
-                    segment_file.write(segment.fullname() + "\n")
+                    name = segment.fullname() + "\n"
+                    name = name[name.startswith(self.remove_prefix) and len(self.remove_prefix) :]
```

Comment:
```suggestion
                    if name.startswith(self.remove_prefix):
                        name = name[len(self.remove_prefix):]
```
This looks easier to read imho.